common/config/api-extractor/api-extractor.json

```diff
@@ -26,6 +26,10 @@
       "ae-forgotten-export": {
         "logLevel": "error",
         "addToApiReportFile": false
+      },
+      "ae-incompatible-release-tags": {
+        "logLevel": "none",
```

Comment:
Do we still want to warn on the command line? Or will that cause build failures as well (because we disallow warnings)?

Reply:
it's currently set to warning and causes build failure.